docs/experiments/prerequisites/event-logging/index.md

```diff
@@ -1,24 +1,179 @@
 # Event logging
 
-The second prerequisite to running experiments on Eppo is logging application events to your warehouse.
+The second prerequisite to running experiments on Eppo is logging application events to your warehouse. It is best practice to centralize application logging as much as possible, and Eppo's SDKs work seamlessly with most logging tools, meaning you can keep using your favorite logger. The last step would be configuring your logging tool, e.g. Segment, to push your data into your data warehouse.
 
-It's best practice to centralize application logging as much as possible. For illustration using a basic web app and Segment as an event logger, you might define the following logging functions on the your client (e.g a single page app):
+Eppo's SDKs include either an assignment logger base class or an interface, in which you can define a method according to your logging requirements. Examples are shown below in Eppo's Node SDK for logging with some common event loggers: 
 
-```js
-// logging.js
+* [Segment](#segment)
+* [Rudderstack](#rudderstack)
+* [mParticle](#mparticle)
+* [Snowplow](#snowplow)
 
-function logEvent(eventData) {
-  segment.track(eventData)
-}
+Similar patterns can be adopted for Eppo's SDKs in other languages, e.g. Java, Python, and etc.
```

Comment:
I'd change to `The examples below are written in Javascript, but similar patterns can be adopted for all of Eppo's SDKs including Java, Python, and etc.` and move below the table.